clients/js/src/embeddings/OpenAIEmbeddingFunction.ts

```diff
@@ -147,10 +147,11 @@ export class OpenAIEmbeddingFunction implements IEmbeddingFunction {
       const { default: openai } = await import("openai");
       // @ts-ignore
       const { VERSION } = await import("openai/version");
+      // @ts-ignore
       return { openai, version: VERSION };
     } catch (e) {
       throw new Error(
-        "Please install openai as a dependency with, e.g. `yarn add openai`",
+        "Please install openai as a dependency with, e.g. `yarn add openai`"
```

Comment:
somewhat minor, but would be nice to unify the error messages in this file with a single package manager (probably npm)

Reply:
I wonder if we even need to put an example command? I feel like people should be able to figure out what we mean when we say `please install the openai npm package` WDYT?

Reply:
seems useful to specify imo 🤷

Reply:
Done! (here and elsewhere)